solarwinds_apm/distro.py

```diff
@@ -229,6 +231,11 @@ def _configure(self, **kwargs):
             OTEL_PYTHON_LOG_FORMAT,
             "%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] [trace_id=%(otelTraceID)s span_id=%(otelSpanID)s trace_flags=%(otelTraceSampled)02d resource.service.name=%(otelServiceName)s] - %(message)s",
         )
+        # Default for ResourceDetector
+        environ.setdefault(
+            OTEL_EXPERIMENTAL_RESOURCE_DETECTORS,
+            ",".join(INTL_SWO_DEFAULT_RESOURCE_DETECTORS),
+        )
```

Comment:
Does this work if `OTEL_EXPERIMENTAL_RESOURCE_DETECTORS` is invalid or oddly formatted? Are invalid entry points skipped and the rest succeed? What if there are duplicates -- is the 2nd attempt to load skipped? Does upstream handle this for us?

* `OTEL_EXPERIMENTAL_RESOURCE_DETECTORS=not-valid,k8s` when `k8s` is already in our defaults
* `OTEL_EXPERIMENTAL_RESOURCE_DETECTORS=,,,`

Reply:
1. Invalid is handled by upstream. In this scenario, k8s will be loaded correctly.  https://github.com/open-telemetry/opentelemetry-python/blob/ffc23bc51956b3cdd8ee5211f6494c37c46d38fe/opentelemetry-sdk/tests/resources/test_resources.py#L806

2. It is considered as empty. https://github.com/open-telemetry/opentelemetry-python/blob/ffc23bc51956b3cdd8ee5211f6494c37c46d38fe/opentelemetry-sdk/tests/resources/test_resources.py#L683